elasticsearch-odfe/config/securityconfig/internal_users.yml

```diff
@@ -0,0 +1,9 @@
+# This is the internal user database
+# The hash value is a bcrypt hash and can be generated with plugin/tools/hash.sh
+
+#password is: admin
+admin:
+  readonly: true
+  hash: $2a$12$VcCDgh2NDk07JGN0rjGbM.Ad41qVR/YFJcgHp0UGns5JDymv..TOG
```

Comment:
It would be nice to have a default one so that the tests can talk to the SSL endpoint without needing to do shananigans to read the password from bats.

Reply:
I moved this to be generated from the default.toml instead, so it's totally overridable for the user. I think that's a better move at least.

Reply:
I think we should take some shenanigans in our tests over shipping a default password.  I would hate to see our users be compromised if this were to be accidentally deployed on a public network without a password change.

Reply:
What shenanigans do you mean? We need to confirm that the securityconfig works and that we can contact the service over https. I'm not sure it gets any simpler than this.

I can do something like echoing unsafe to the CLI perhaps?

Reply:
I had thought postgres had the behavior of generating a password if one wasn't set and not shipping with a default, but it looks like it ships with `admin/admin`. That still feels very unsafe to me but I think that we can revisit this topic in a broader scope of guidelines for default credentials in coreplans. Consider this request revoked for this PR